docs/README.md

```diff
@@ -33,7 +56,7 @@ Sitemap generation already in place
 Next generation available: <End-date>	
 ```	
 
-*To make a force restart, add the `__force` query string at the end of the API URL, and send a new request. But, be aware that this will cancel the previous process.*	
+*To make a force restart, add the `force` argument to the query like: `generateSitemap(force: true)`. But, be aware that this will cancel the previous process.*	
```

Comment:
```suggestion
*To make a force restart, add the `force` argument to the query, as in: `generateSitemap(force: true)`. But, be aware that this will cancel the previous process.*	
```